Autocoders/Python/src/fprime_ac/generators/templates/test/test_main.tmpl

```diff
@@ -6,7 +6,11 @@
 
 #for $test_name, $method_name in $test_cases
 TEST(Nominal, ${test_name}) {
-    Ref::Tester tester;
+    #if $namespace_list != None
+    #echo "::".join($namespace_list)#::Tester tester;
+    #else
+    Tester tester;
```

Comment:
Are there legitimate cases where the namespace_list could be empty/none? Not that the handling should be different, just wondering.

Reply:
I don't believe it is possible, but it is correct to have it implemented in the case I am mistaken.